apps/src/templates/LoadableComponent.jsx

```diff
@@ -85,7 +85,7 @@ LoadableComponent.propTypes = {
   /** Optional. Additional styles to apply to the loading spinner. */
   spinnerStyle: PropTypes.object,
   /** Optional. Size of the loading spinner. Defaults to `medium` */
-  spinnerSize: PropTypes.oneOf('small', 'medium', 'large'),
+  spinnerSize: PropTypes.oneOf(['small', 'medium', 'large']),
```

Comment:
There was a warning about this when I was debugging a bunch of stuff in storybook. Just took care of it while I was there